source/developing.html.erb.md

```diff
@@ -111,7 +111,7 @@ Provide an indication of language in the `html` tag using the `lang` attribute.
 {:.attach_permalink}
 ## Use mark-up to convey meaning and structure
 
-Use structural elements to mark-up headings, lists, tables, etc. HTML5 provides additional elements, such as `<nav>` and `<aside>`, to better structure your documents. <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> roles can provide additional meaning, for example, using `role='search'` to identify search functionality. Work with designers and content writers to agree on meanings and then use them consistently.
+Mark-up headings, lists, tables, and other page structures. HTML5 provides additional elements, such as `<nav>` and `<aside>`, to better structure your content. <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> roles can provide additional meaning, for example, using `role='search'` to identify search functionality. Work with designers and content writers to agree on meanings and then use them consistently.
```

Comment:
I have a medium concern that the first sentence isn't necessarily as precise as it could be. With that advice, marking up a heading as:

```html
<div class="heading-1">This is a heading</div>
```

is feasible, and meets the advice. Perhaps something such as:

> Use appropriate mark-up for headings, lists, tables, etc.

Also, not 100% on what 'other page structures' might be. It could overlap with the second sentence and cause a bit of confusion.